chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/auth/UserPermissionDTO.java

```diff
@@ -1,8 +1,11 @@
 package gov.healthit.chpl.dto.auth;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
 import gov.healthit.chpl.auth.permission.GrantedPermission;
 import gov.healthit.chpl.entity.auth.UserPermissionEntity;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Does the UserDTO object not need this same annotation? It may not if the code is able to deserialize all the old user activities but I just wasn't sure. Maybe we haven't changed that object much over time.

Reply:
I did some checking with older activities and there does not appear to be any problems.